docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1647 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    _ = gc.collect()
</pre><p>Although I think I know what you're doing, this is kind of an odd thing to have to do at all. It's not the kind of thing you would necessarily write as part of a normal analysis I think.</p><p> </p><p>Does it make sense to instead define the benchmarks inside functions that set up the data, run, and collate results. Then the dataframes don't escape the scope of the function call and will get collected automatically.</p><p> </p><p>In particular, the requirement to do <code>gc.collect()</code> (presumably to clean out cycles) is an anti-pattern that readers of this notebook may well cargo-cult. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>